wiki/Modding_Guides/Getting_Your_Map_Modded/de.md

```diff
@@ -26,7 +29,7 @@ Ein Mod4Mod, oder auch M4M abgekürzt ist eine Methode wo du die Map eines ander
 Direktes fragen anderer Spieler
 -------------------------------
 
-Auch das direkte fragen anderer Spieler ist eine sehr effektive Methode um an Mods zu kommen. Sucht nacht Spielern mit vielen Kudosu aber keiner Modding Queue und fragt diese direkt. Eine gute Idee ist es auch Spieler mit einem Hohen Rang nach einem "Testplay" zu fragen, oftmals geben diese dann Verbesserungsvorschläge was man an der Map besser machen könnte. Auch das direkte fragen von Mitgliedern der [Beatmap Nominator Gruppe](DE:BNG "wikilink") ist eine gute Idee. Hier sind noch ein paar Tipps die man beachten sollte wenn man Spieler direkt fragt:
+Auch das direkte fragen anderer Spieler ist eine sehr effektive Methode um an Mods zu kommen. Sucht nacht Spielern mit vielen Kudosu aber keiner Modding Queue und fragt diese direkt. Eine gute Idee ist es auch Spieler mit einem Hohen Rang nach einem "Testplay" zu fragen, oftmals geben diese dann Verbesserungsvorschläge was man an der Map besser machen könnte. Auch das direkte fragen von Mitgliedern der [Beatmap Nominator Gruppe](DE:/wiki/People/Beatmap_Nomination_Group) ist eine gute Idee. Hier sind noch ein paar Tipps die man beachten sollte wenn man Spieler direkt fragt:
```

Comment:
whoops, I had meant to give the correction.. I guess osu-web took my attention from this review comment :/